internal/tags/expand.go

```diff
@@ -11,3 +11,15 @@ func Expand(tagsMap map[string]interface{}) map[string]*string {
 
 	return output
 }
+
+func ExpandPandora(tagsMap map[string]interface{}) map[string]string {
+	output := make(map[string]string, len(tagsMap))
+
+	for i, v := range tagsMap {
+		// Validate should have ignored this error already
+		value, _ := TagValueToString(v)
+		output[i] = value
+	}
+
+	return output
+}
```

Comment:
we're intentionally duplicating the Expand/Flatten functions into `transition.go` in each Service Package for the moment - since this `tags` package will be removed in the not-too-distant future (in place of updating the `tags` package within `go-azure-helpers` to account for the updated Tags structure) - so can we do that here?

```suggestion
```

Reply:
for now you can copy this file into the specific service package: https://github.com/hashicorp/terraform-provider-azurerm/blob/51e6eb72f6e5c57cdd774d85b20cedb927a0d178/internal/services/msi/transition.go#L1

whilst that's (intentional) tech-debt - we'll go through and clean those up in the near future, so it's fine for now 👍